cmd/tsbs_generate_queries/main.go

```diff
@@ -79,10 +79,11 @@ func getGenerator(format string, start, end time.Time, scale int) utils.DevopsGe
 		return mongo.NewDevops(start, end, scale)
 	} else if format == "mongo-naive" {
 		return mongo.NewNaiveDevops(start, end, scale)
-	} else if format == "timescaledb" {
+	} else if format == "timescaledb" || format == "postgresql" {
 		tgen := timescaledb.NewDevops(start, end, scale)
 		tgen.UseJSON = timescaleUseJSON
 		tgen.UseTags = timescaleUseTags
+		tgen.UseTimeBucket = format == "timescaledb"
```

Comment:
Maybe there should be a `timescaleUseTimeBucket` flag as well so users can benchmark TimescaleDB w/o time_bucket (not sure why you'd want to, but who knows). You could have it default `true` and then this would become `format == "timescaledb" && timescaleUseTimeBucket`

Reply:
I think we could keep both, but if you don't want to add the format to `generate_data` (which I am still on the fence about), I think it's fine to remove the postgresql format. We can update the README / documentation to make it clear that you need to set `--timescale-use-time-bucket=false` in order to test on PostgreSQL. I think that's reasonable, actually.

Reply:
Makes sense, will update the PR accordingly. Just let me know if you change your opinion wrt `generate_data`.